reconcile/change_owners/change_owners.py

```diff
@@ -370,7 +378,7 @@ def run(
         labels = manage_conditional_label(
             labels=labels,
             condition=self_serviceable and approved,
-            true_label=APPROVED,
+            true_label=get_approved_label_by_priority(change_type_processors),
```

Comment:
have a look at the implementation of `manage_conditional_label` and think about the following scenario:

* MR is opened with involved two change-types, one `medium` and one `critical`
* the `medium` label is applied
* the tenant changes the MR and removes the change that was covered by the `medium` change-type